src/mozilla_sec_eia/models/sec10k/utils/cloud.py

```diff
@@ -29,6 +29,20 @@ def _compute_md5(file_path: Path) -> str:
     return base64.b64encode(hash_md5.digest()).decode()
 
 
+def convert_ex21_id_to_filename(df: pd.DataFrame, id_col_name: str = "id"):
```

Comment:
There's still a discrepancy in how the filenames are formatted in the Ex. 21 table and how they're formatted in the metadata. I added this helper function so convert between the two formats. But should probably just handle this at the source of the discrepancy. Also not sure if this helper function should really live in `cloud.py`